react/features/facial-recognition/actions.js

```diff
@@ -52,80 +59,97 @@ let duplicateConsecutiveExpressions = 0;
 /**
  * Variable that keeps the interval for sending expressions to webhook.
  */
-let sendInterval;
+let webhookSendInterval;
+
+/**
+ * Variable that keeps the interval for detecting faces in a frame.
+ */
+let detectionInterval;
 
 /**
  * Loads the worker that predicts the facial expression.
  *
  * @returns {void}
  */
 export function loadWorker() {
-    return function(dispatch: Function) {
-        if (!window.Worker) {
-            logger.warn('Browser does not support web workers');
+    return function(dispatch: Function, getState: Function) {
+        if (navigator.product === 'ReactNative') {
+            logger.warn('Unsupported environment for face recognition');
 
             return;
         }
 
-        const baseUrl = `${getBaseUrl()}/libs/`;
+        const baseUrl = `${getBaseUrl()}libs/`;
         let workerUrl = `${baseUrl}facial-expressions-worker.min.js`;
 
         const workerBlob = new Blob([ `importScripts("${workerUrl}");` ], { type: 'application/javascript' });
 
         workerUrl = window.URL.createObjectURL(workerBlob);
-        worker = new Worker(workerUrl, { name: 'Facial Expression Worker' });
+        worker = new Worker(workerUrl, { name: 'Face Recognition Worker' });
```

Comment:
Let's rename this whole feature please. We chose a poor name which is alarming for some, since we are not recognizing faces. We are detecting landmarks. The feature can be "face-landmarks" and the worker friendly name "Face Landmark Detection Worker".

Reply:
will do so in a future pr, since it will touch other repos, and it will make this one very hard to follow / review

Reply:
Git is quite smart if you rename files, but ok. What other repos do we need to modify?

Reply:
problem is we also need to modify variable names, functions, comments, etc